src/matrix_operations.cpp

```diff
@@ -44,3 +44,86 @@ double max_offdiag_symmetric(const arma::mat& A, int& k, int& l)
     }
     return max;
 }
+
```

Comment:
Dette synes jeg er vanskelig med branches og pull requests i git, men jeg tror det er et godt mål å ikke gjøre de samme endringene i flere branches. Iallfall ikke pushe det. Noe som av og til funker er å kopiere inn de endringene du tar utgangspunkt i, men uten å committe de. Jeg synes det ofte kan være enklere

Reply:
Flott, takk for tips! Da gjør jeg det neste gang:)